ImageConverter/ImageConverter/Ppm/PpmImageWriter.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace ImageConverter.Ppm;
+
+public class PpmImageWriter : IImageWriter
+{
+    private const string fileFormatNumber = "P3";
```

Comment:
Why use constant here but plain string parameter in reader?

Reply:
add this variable to reader in 790fca6